frontend/sass/_forms.scss

```diff
@@ -15,6 +15,11 @@ textarea {
   border: 3px solid #ccc;
 }
 
+input[type="url"]
```

Comment:
Just move the `input[type="url"]` target up to line `14`. You don't need to recopy the `textarea` target and style rule. So, final result, starting at current line 13 should be:

```css
input[type="text"],
input[type="url"],
textarea {
  border: 3px solid #ccc;
}
```